### PR TITLE
Document pm.environment.name

### PR DESCRIPTION
**Pull request by arlemi**
Adds `pm.environment.name` to the documentation in the Postman -> Scripts -> Postman Sandbox section. See #1922. 
Edit: Added updates to the Postman Sandbox API References too

### CODE REVIEW

**Comment by SueSmith** [comment]
Thanks for this, I'd love to revamp how we document this stuff fairly soon. For now let's just aim to keep it consistent - will add a wee suggestion comment inline.

**Comment by arlemi** [comment]
Last commit adds the variable type on both pages. There seems to be two different conventions for it though (`variable:type` for [Postman Sandbox](https://learning.getpostman.com/docs/postman/scripts/postman-sandbox/) and `variable {type}` for [Postman Sandbox API Reference](https://learning.getpostman.com/docs/postman/scripts/postman-sandbox-api-reference/)) so I used both to keep it consistent with the page it was on.

File: src/pages/docs/postman/scripts/postman-sandbox-api-reference.md [hunk 1]
```diff
@@ -171,6 +171,7 @@ The variables defined in the individual scopes may also be accessed via `pm.envi
 
 `pm.environment:` [Read more about VariableScope](http://www.postmanlabs.com/postman-collection/VariableScope.html)
 
+* `pm.environment.name`: Contains the name of the current environment.
```
[the remark this answers is not in the document]
Comment (arlemi) [reply src/pages/docs/postman/scripts/postman-sandbox-api-reference.md]: My bad! Should be fixed now.

**Comment by SueSmith** [comment]
Ahh good to know, will sort that during redrafts. Thanks!

**Pull request merged**